Framework/PythonInterface/mantid/api/src/Exports/DetectorInfo.cpp

```diff
@@ -0,0 +1,14 @@
+#include "MantidAPI/DetectorInfo.h"
+#include <boost/python/class.hpp>
+
+using Mantid::API::DetectorInfo;
+using namespace boost::python;
+
+void export_DetectorInfo() {
+  class_<DetectorInfo, boost::noncopyable>("DetectorInfo", no_init)
+      .def("size", &DetectorInfo::size, (arg("self")),
```

Comment:
If we rename `::size` then I think it should be on both the Python and C++ side to match. I don't see a compelling reason for them to be different.